controller_manager/test/test_controller_manager_srvs.cpp

```diff
@@ -0,0 +1,333 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
```suggestion
// Copyright 2020 PAL Robotics S.L.
```

This is definitely all new